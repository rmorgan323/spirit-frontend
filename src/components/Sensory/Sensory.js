import React from 'react';
import SessionHeader from '../SessionHeader/SessionHeader';
import CategorySliders from '../CategorySliders/CategorySliders';
import { sensory } from '../../language/language';
import './Sensory.css';

const Sensory = () => {
  const { header, columns } = sensory;

  return (
    <div className="Sensory">
      <SessionHeader
        key={header.db}
        title={header.title}
        headings={header.terms}
        databaseNamesArray={header.db}
      />

      <div className="category-content">
        {Object.keys(columns).map(column => {
          const { title, terms, db } = columns[column];

          return (
            <CategorySliders
              key={db}
              title={title}
              slidersArray={terms}
              databaseNamesArray={db}
            />
          );
        })}
      </div>
    </div>
  );
};

export default Sensory;
