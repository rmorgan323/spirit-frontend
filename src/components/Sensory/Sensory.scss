 .Sensory {

  .category-content {
    display: flex;
  }
 }