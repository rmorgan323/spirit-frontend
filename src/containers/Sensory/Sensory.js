import React, { Component } from 'react';
import SessionHeader from '../SessionHeader/SessionHeader';
import CategorySliders from '../../components/CategorySliders/CategorySliders';
import './Sensory.css';

class Sensory extends Component {
  constructor() {
    super();

    this.state = {
    }
  }

  render() {
    return (
      <div className="Sensory">
        <SessionHeader 
          title={'Sensory Discrimination Capacities'}
          headings={['Vestibular', 'Proprioception', 'Tactile', 'Auditory', 'Visual', 'Intero']}
        />
        <div className="category-content">
          <CategorySliders 
            title={'Sensory Discrimination / Perceptual Skills'}
            slidersArray={['Ability to Recognize', 'Match', 'Classification', 'Spatial Organization', 'Figure Ground Perception', 'Inferences and Abstractions (logic)']}
          />
          <CategorySliders 
            title={'Ideation'}
            slidersArray={['Initiation', 'Imitation (mimic or emulate?)', 'Affordance-based (obvious, imitative or expands?)', 'Generativity']}
          />
          <CategorySliders 
            title={'Planning'}
            slidersArray={['Sequencing', 'Sequencing/Gestalt', 'Schema Selection', 'Schema Expansion', 'Modification']}
          />
          <CategorySliders 
            title={'Execution'}
            slidersArray={['Timing', 'Timing/Force', 'Spatial', 'Temporal', 'Control (Postural; FF; FB)', 'Monitoring']}
          />
          <CategorySliders 
            slidersArray={['Oral', 'Ocular', 'Postural', 'Limb', 'Gross', 'Fine', 'Tool/Object', 'Projected Action Sequences']}
          />
          <CategorySliders 
            title={"Ayres' Factors"}
            slidersArray={['Somatodyspraxia', 'Visual Dyspraxia', 'Generalized Dyspraxis', 'BMC/Balance/Sequencing/ML', 'Praxis on Verbal Command']}
          />
        </div>
      </div>
    )
  }
}

export default Sensory;