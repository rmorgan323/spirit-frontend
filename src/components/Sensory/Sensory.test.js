import React from 'react';
import Sensory from './Sensory';
import { shallow } from 'enzyme';

describe('Sensory tests', () => {
  it('Should match snapshot', () => {
    const renderedApp = shallow(<Sensory />);

    expect(renderedApp).toMatchSnapshot();
  });
});
