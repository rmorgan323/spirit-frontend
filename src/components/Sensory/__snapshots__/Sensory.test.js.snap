// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sensory tests Should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Sensory />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <SessionHeader
          databaseNamesArray={
            Array [
              "sen_h_vestibular",
              "sen_h_proprioception",
              "sen_h_tactile",
              "sen_h_auditory",
              "sen_h_visual",
              "sen_h_intero",
            ]
          }
          headings={
            Array [
              "Vestibular",
              "Proprioception",
              "Tactile",
              "Auditory",
              "Visual",
              "Intero",
            ]
          }
          title="Sensory Discrimination Capacities"
        />,
        <div
          className="category-content"
        >
          <CategorySliders
            databaseNamesArray={
              Array [
                "sen_1_ability",
                "sen_1_match",
                "sen_1_classification",
                "sen_1_spatial",
                "sen_1_figure",
                "sen_1_inferences",
              ]
            }
            slidersArray={
              Array [
                "Ability to Recognize",
                "Match",
                "Classification",
                "Spatial Organization",
                "Figure Ground Perception",
                "Inferences and Abstractions (logic)",
              ]
            }
            title="Sensory Discrimination / Perceptual Skills"
          />
          <CategorySliders
            databaseNamesArray={
              Array [
                "sen_2_initiation",
                "sen_2_imitation",
                "sen_2_affordance",
                "sen_2_generativity",
              ]
            }
            slidersArray={
              Array [
                "Initiation",
                "Imitation (mimic or emulate?)",
                "Affordance-based (obvious, imitative or expands?)",
                "Generativity",
              ]
            }
            title="Ideation"
          />
          <CategorySliders
            databaseNamesArray={
              Array [
                "sen_3_sequencing",
                "sen_3_sequencing_gestalt",
                "sen_3_schema_selection",
                "sen_3_schema_expansion",
                "sen_3_modification",
              ]
            }
            slidersArray={
              Array [
                "Sequencing",
                "Sequencing/Gestalt",
                "Schema Selection",
                "Schema Expansion",
                "Modification",
              ]
            }
            title="Planning"
          />
          <CategorySliders
            databaseNamesArray={
              Array [
                "sen_4_timing",
                "sen_4_timing_force",
                "sen_4_spatial",
                "sen_4_temporal",
                "sen_4_control",
                "sen_4_monitoring",
              ]
            }
            slidersArray={
              Array [
                "Timing",
                "Timing/Force",
                "Spatial",
                "Temporal",
                "Control (Postural; FF; FB)",
                "Monitoring",
              ]
            }
            title="Execution"
          />
          <CategorySliders
            databaseNamesArray={
              Array [
                "sen_5_oral",
                "sen_5_ocular",
                "sen_5_postural",
                "sen_5_limb",
                "sen_5_gross",
                "sen_5_fine",
                "sen_5_tool",
                "sen_5_projected",
              ]
            }
            slidersArray={
              Array [
                "Oral",
                "Ocular",
                "Postural",
                "Limb",
                "Gross",
                "Fine",
                "Tool/Object",
                "Projected Action Sequences",
              ]
            }
            title="Skill Domain"
          />
          <CategorySliders
            databaseNamesArray={
              Array [
                "sen_6_somatodyspraxia",
                "sen_6_visual",
                "sen_6_generalized",
                "sen_6_bmc",
                "sen_6_praxis",
              ]
            }
            slidersArray={
              Array [
                "Somatodyspraxia",
                "Visual Dyspraxia",
                "Generalized Dyspraxis",
                "BMC/Balance/Sequencing/ML",
                "Praxis on Verbal Command",
              ]
            }
            title="Ayres' Factors"
          />
        </div>,
      ],
      "className": "Sensory",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": "sen_h_vestibular,sen_h_proprioception,sen_h_tactile,sen_h_auditory,sen_h_visual,sen_h_intero",
        "nodeType": "function",
        "props": Object {
          "databaseNamesArray": Array [
            "sen_h_vestibular",
            "sen_h_proprioception",
            "sen_h_tactile",
            "sen_h_auditory",
            "sen_h_visual",
            "sen_h_intero",
          ],
          "headings": Array [
            "Vestibular",
            "Proprioception",
            "Tactile",
            "Auditory",
            "Visual",
            "Intero",
          ],
          "title": "Sensory Discrimination Capacities",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <CategorySliders
              databaseNamesArray={
                Array [
                  "sen_1_ability",
                  "sen_1_match",
                  "sen_1_classification",
                  "sen_1_spatial",
                  "sen_1_figure",
                  "sen_1_inferences",
                ]
              }
              slidersArray={
                Array [
                  "Ability to Recognize",
                  "Match",
                  "Classification",
                  "Spatial Organization",
                  "Figure Ground Perception",
                  "Inferences and Abstractions (logic)",
                ]
              }
              title="Sensory Discrimination / Perceptual Skills"
            />,
            <CategorySliders
              databaseNamesArray={
                Array [
                  "sen_2_initiation",
                  "sen_2_imitation",
                  "sen_2_affordance",
                  "sen_2_generativity",
                ]
              }
              slidersArray={
                Array [
                  "Initiation",
                  "Imitation (mimic or emulate?)",
                  "Affordance-based (obvious, imitative or expands?)",
                  "Generativity",
                ]
              }
              title="Ideation"
            />,
            <CategorySliders
              databaseNamesArray={
                Array [
                  "sen_3_sequencing",
                  "sen_3_sequencing_gestalt",
                  "sen_3_schema_selection",
                  "sen_3_schema_expansion",
                  "sen_3_modification",
                ]
              }
              slidersArray={
                Array [
                  "Sequencing",
                  "Sequencing/Gestalt",
                  "Schema Selection",
                  "Schema Expansion",
                  "Modification",
                ]
              }
              title="Planning"
            />,
            <CategorySliders
              databaseNamesArray={
                Array [
                  "sen_4_timing",
                  "sen_4_timing_force",
                  "sen_4_spatial",
                  "sen_4_temporal",
                  "sen_4_control",
                  "sen_4_monitoring",
                ]
              }
              slidersArray={
                Array [
                  "Timing",
                  "Timing/Force",
                  "Spatial",
                  "Temporal",
                  "Control (Postural; FF; FB)",
                  "Monitoring",
                ]
              }
              title="Execution"
            />,
            <CategorySliders
              databaseNamesArray={
                Array [
                  "sen_5_oral",
                  "sen_5_ocular",
                  "sen_5_postural",
                  "sen_5_limb",
                  "sen_5_gross",
                  "sen_5_fine",
                  "sen_5_tool",
                  "sen_5_projected",
                ]
              }
              slidersArray={
                Array [
                  "Oral",
                  "Ocular",
                  "Postural",
                  "Limb",
                  "Gross",
                  "Fine",
                  "Tool/Object",
                  "Projected Action Sequences",
                ]
              }
              title="Skill Domain"
            />,
            <CategorySliders
              databaseNamesArray={
                Array [
                  "sen_6_somatodyspraxia",
                  "sen_6_visual",
                  "sen_6_generalized",
                  "sen_6_bmc",
                  "sen_6_praxis",
                ]
              }
              slidersArray={
                Array [
                  "Somatodyspraxia",
                  "Visual Dyspraxia",
                  "Generalized Dyspraxis",
                  "BMC/Balance/Sequencing/ML",
                  "Praxis on Verbal Command",
                ]
              }
              title="Ayres' Factors"
            />,
          ],
          "className": "category-content",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": "sen_1_ability,sen_1_match,sen_1_classification,sen_1_spatial,sen_1_figure,sen_1_inferences",
            "nodeType": "function",
            "props": Object {
              "databaseNamesArray": Array [
                "sen_1_ability",
                "sen_1_match",
                "sen_1_classification",
                "sen_1_spatial",
                "sen_1_figure",
                "sen_1_inferences",
              ],
              "slidersArray": Array [
                "Ability to Recognize",
                "Match",
                "Classification",
                "Spatial Organization",
                "Figure Ground Perception",
                "Inferences and Abstractions (logic)",
              ],
              "title": "Sensory Discrimination / Perceptual Skills",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": "sen_2_initiation,sen_2_imitation,sen_2_affordance,sen_2_generativity",
            "nodeType": "function",
            "props": Object {
              "databaseNamesArray": Array [
                "sen_2_initiation",
                "sen_2_imitation",
                "sen_2_affordance",
                "sen_2_generativity",
              ],
              "slidersArray": Array [
                "Initiation",
                "Imitation (mimic or emulate?)",
                "Affordance-based (obvious, imitative or expands?)",
                "Generativity",
              ],
              "title": "Ideation",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": "sen_3_sequencing,sen_3_sequencing_gestalt,sen_3_schema_selection,sen_3_schema_expansion,sen_3_modification",
            "nodeType": "function",
            "props": Object {
              "databaseNamesArray": Array [
                "sen_3_sequencing",
                "sen_3_sequencing_gestalt",
                "sen_3_schema_selection",
                "sen_3_schema_expansion",
                "sen_3_modification",
              ],
              "slidersArray": Array [
                "Sequencing",
                "Sequencing/Gestalt",
                "Schema Selection",
                "Schema Expansion",
                "Modification",
              ],
              "title": "Planning",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": "sen_4_timing,sen_4_timing_force,sen_4_spatial,sen_4_temporal,sen_4_control,sen_4_monitoring",
            "nodeType": "function",
            "props": Object {
              "databaseNamesArray": Array [
                "sen_4_timing",
                "sen_4_timing_force",
                "sen_4_spatial",
                "sen_4_temporal",
                "sen_4_control",
                "sen_4_monitoring",
              ],
              "slidersArray": Array [
                "Timing",
                "Timing/Force",
                "Spatial",
                "Temporal",
                "Control (Postural; FF; FB)",
                "Monitoring",
              ],
              "title": "Execution",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": "sen_5_oral,sen_5_ocular,sen_5_postural,sen_5_limb,sen_5_gross,sen_5_fine,sen_5_tool,sen_5_projected",
            "nodeType": "function",
            "props": Object {
              "databaseNamesArray": Array [
                "sen_5_oral",
                "sen_5_ocular",
                "sen_5_postural",
                "sen_5_limb",
                "sen_5_gross",
                "sen_5_fine",
                "sen_5_tool",
                "sen_5_projected",
              ],
              "slidersArray": Array [
                "Oral",
                "Ocular",
                "Postural",
                "Limb",
                "Gross",
                "Fine",
                "Tool/Object",
                "Projected Action Sequences",
              ],
              "title": "Skill Domain",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": "sen_6_somatodyspraxia,sen_6_visual,sen_6_generalized,sen_6_bmc,sen_6_praxis",
            "nodeType": "function",
            "props": Object {
              "databaseNamesArray": Array [
                "sen_6_somatodyspraxia",
                "sen_6_visual",
                "sen_6_generalized",
                "sen_6_bmc",
                "sen_6_praxis",
              ],
              "slidersArray": Array [
                "Somatodyspraxia",
                "Visual Dyspraxia",
                "Generalized Dyspraxis",
                "BMC/Balance/Sequencing/ML",
                "Praxis on Verbal Command",
              ],
              "title": "Ayres' Factors",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <SessionHeader
            databaseNamesArray={
              Array [
                "sen_h_vestibular",
                "sen_h_proprioception",
                "sen_h_tactile",
                "sen_h_auditory",
                "sen_h_visual",
                "sen_h_intero",
              ]
            }
            headings={
              Array [
                "Vestibular",
                "Proprioception",
                "Tactile",
                "Auditory",
                "Visual",
                "Intero",
              ]
            }
            title="Sensory Discrimination Capacities"
          />,
          <div
            className="category-content"
          >
            <CategorySliders
              databaseNamesArray={
                Array [
                  "sen_1_ability",
                  "sen_1_match",
                  "sen_1_classification",
                  "sen_1_spatial",
                  "sen_1_figure",
                  "sen_1_inferences",
                ]
              }
              slidersArray={
                Array [
                  "Ability to Recognize",
                  "Match",
                  "Classification",
                  "Spatial Organization",
                  "Figure Ground Perception",
                  "Inferences and Abstractions (logic)",
                ]
              }
              title="Sensory Discrimination / Perceptual Skills"
            />
            <CategorySliders
              databaseNamesArray={
                Array [
                  "sen_2_initiation",
                  "sen_2_imitation",
                  "sen_2_affordance",
                  "sen_2_generativity",
                ]
              }
              slidersArray={
                Array [
                  "Initiation",
                  "Imitation (mimic or emulate?)",
                  "Affordance-based (obvious, imitative or expands?)",
                  "Generativity",
                ]
              }
              title="Ideation"
            />
            <CategorySliders
              databaseNamesArray={
                Array [
                  "sen_3_sequencing",
                  "sen_3_sequencing_gestalt",
                  "sen_3_schema_selection",
                  "sen_3_schema_expansion",
                  "sen_3_modification",
                ]
              }
              slidersArray={
                Array [
                  "Sequencing",
                  "Sequencing/Gestalt",
                  "Schema Selection",
                  "Schema Expansion",
                  "Modification",
                ]
              }
              title="Planning"
            />
            <CategorySliders
              databaseNamesArray={
                Array [
                  "sen_4_timing",
                  "sen_4_timing_force",
                  "sen_4_spatial",
                  "sen_4_temporal",
                  "sen_4_control",
                  "sen_4_monitoring",
                ]
              }
              slidersArray={
                Array [
                  "Timing",
                  "Timing/Force",
                  "Spatial",
                  "Temporal",
                  "Control (Postural; FF; FB)",
                  "Monitoring",
                ]
              }
              title="Execution"
            />
            <CategorySliders
              databaseNamesArray={
                Array [
                  "sen_5_oral",
                  "sen_5_ocular",
                  "sen_5_postural",
                  "sen_5_limb",
                  "sen_5_gross",
                  "sen_5_fine",
                  "sen_5_tool",
                  "sen_5_projected",
                ]
              }
              slidersArray={
                Array [
                  "Oral",
                  "Ocular",
                  "Postural",
                  "Limb",
                  "Gross",
                  "Fine",
                  "Tool/Object",
                  "Projected Action Sequences",
                ]
              }
              title="Skill Domain"
            />
            <CategorySliders
              databaseNamesArray={
                Array [
                  "sen_6_somatodyspraxia",
                  "sen_6_visual",
                  "sen_6_generalized",
                  "sen_6_bmc",
                  "sen_6_praxis",
                ]
              }
              slidersArray={
                Array [
                  "Somatodyspraxia",
                  "Visual Dyspraxia",
                  "Generalized Dyspraxis",
                  "BMC/Balance/Sequencing/ML",
                  "Praxis on Verbal Command",
                ]
              }
              title="Ayres' Factors"
            />
          </div>,
        ],
        "className": "Sensory",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": "sen_h_vestibular,sen_h_proprioception,sen_h_tactile,sen_h_auditory,sen_h_visual,sen_h_intero",
          "nodeType": "function",
          "props": Object {
            "databaseNamesArray": Array [
              "sen_h_vestibular",
              "sen_h_proprioception",
              "sen_h_tactile",
              "sen_h_auditory",
              "sen_h_visual",
              "sen_h_intero",
            ],
            "headings": Array [
              "Vestibular",
              "Proprioception",
              "Tactile",
              "Auditory",
              "Visual",
              "Intero",
            ],
            "title": "Sensory Discrimination Capacities",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <CategorySliders
                databaseNamesArray={
                  Array [
                    "sen_1_ability",
                    "sen_1_match",
                    "sen_1_classification",
                    "sen_1_spatial",
                    "sen_1_figure",
                    "sen_1_inferences",
                  ]
                }
                slidersArray={
                  Array [
                    "Ability to Recognize",
                    "Match",
                    "Classification",
                    "Spatial Organization",
                    "Figure Ground Perception",
                    "Inferences and Abstractions (logic)",
                  ]
                }
                title="Sensory Discrimination / Perceptual Skills"
              />,
              <CategorySliders
                databaseNamesArray={
                  Array [
                    "sen_2_initiation",
                    "sen_2_imitation",
                    "sen_2_affordance",
                    "sen_2_generativity",
                  ]
                }
                slidersArray={
                  Array [
                    "Initiation",
                    "Imitation (mimic or emulate?)",
                    "Affordance-based (obvious, imitative or expands?)",
                    "Generativity",
                  ]
                }
                title="Ideation"
              />,
              <CategorySliders
                databaseNamesArray={
                  Array [
                    "sen_3_sequencing",
                    "sen_3_sequencing_gestalt",
                    "sen_3_schema_selection",
                    "sen_3_schema_expansion",
                    "sen_3_modification",
                  ]
                }
                slidersArray={
                  Array [
                    "Sequencing",
                    "Sequencing/Gestalt",
                    "Schema Selection",
                    "Schema Expansion",
                    "Modification",
                  ]
                }
                title="Planning"
              />,
              <CategorySliders
                databaseNamesArray={
                  Array [
                    "sen_4_timing",
                    "sen_4_timing_force",
                    "sen_4_spatial",
                    "sen_4_temporal",
                    "sen_4_control",
                    "sen_4_monitoring",
                  ]
                }
                slidersArray={
                  Array [
                    "Timing",
                    "Timing/Force",
                    "Spatial",
                    "Temporal",
                    "Control (Postural; FF; FB)",
                    "Monitoring",
                  ]
                }
                title="Execution"
              />,
              <CategorySliders
                databaseNamesArray={
                  Array [
                    "sen_5_oral",
                    "sen_5_ocular",
                    "sen_5_postural",
                    "sen_5_limb",
                    "sen_5_gross",
                    "sen_5_fine",
                    "sen_5_tool",
                    "sen_5_projected",
                  ]
                }
                slidersArray={
                  Array [
                    "Oral",
                    "Ocular",
                    "Postural",
                    "Limb",
                    "Gross",
                    "Fine",
                    "Tool/Object",
                    "Projected Action Sequences",
                  ]
                }
                title="Skill Domain"
              />,
              <CategorySliders
                databaseNamesArray={
                  Array [
                    "sen_6_somatodyspraxia",
                    "sen_6_visual",
                    "sen_6_generalized",
                    "sen_6_bmc",
                    "sen_6_praxis",
                  ]
                }
                slidersArray={
                  Array [
                    "Somatodyspraxia",
                    "Visual Dyspraxia",
                    "Generalized Dyspraxis",
                    "BMC/Balance/Sequencing/ML",
                    "Praxis on Verbal Command",
                  ]
                }
                title="Ayres' Factors"
              />,
            ],
            "className": "category-content",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": "sen_1_ability,sen_1_match,sen_1_classification,sen_1_spatial,sen_1_figure,sen_1_inferences",
              "nodeType": "function",
              "props": Object {
                "databaseNamesArray": Array [
                  "sen_1_ability",
                  "sen_1_match",
                  "sen_1_classification",
                  "sen_1_spatial",
                  "sen_1_figure",
                  "sen_1_inferences",
                ],
                "slidersArray": Array [
                  "Ability to Recognize",
                  "Match",
                  "Classification",
                  "Spatial Organization",
                  "Figure Ground Perception",
                  "Inferences and Abstractions (logic)",
                ],
                "title": "Sensory Discrimination / Perceptual Skills",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": "sen_2_initiation,sen_2_imitation,sen_2_affordance,sen_2_generativity",
              "nodeType": "function",
              "props": Object {
                "databaseNamesArray": Array [
                  "sen_2_initiation",
                  "sen_2_imitation",
                  "sen_2_affordance",
                  "sen_2_generativity",
                ],
                "slidersArray": Array [
                  "Initiation",
                  "Imitation (mimic or emulate?)",
                  "Affordance-based (obvious, imitative or expands?)",
                  "Generativity",
                ],
                "title": "Ideation",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": "sen_3_sequencing,sen_3_sequencing_gestalt,sen_3_schema_selection,sen_3_schema_expansion,sen_3_modification",
              "nodeType": "function",
              "props": Object {
                "databaseNamesArray": Array [
                  "sen_3_sequencing",
                  "sen_3_sequencing_gestalt",
                  "sen_3_schema_selection",
                  "sen_3_schema_expansion",
                  "sen_3_modification",
                ],
                "slidersArray": Array [
                  "Sequencing",
                  "Sequencing/Gestalt",
                  "Schema Selection",
                  "Schema Expansion",
                  "Modification",
                ],
                "title": "Planning",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": "sen_4_timing,sen_4_timing_force,sen_4_spatial,sen_4_temporal,sen_4_control,sen_4_monitoring",
              "nodeType": "function",
              "props": Object {
                "databaseNamesArray": Array [
                  "sen_4_timing",
                  "sen_4_timing_force",
                  "sen_4_spatial",
                  "sen_4_temporal",
                  "sen_4_control",
                  "sen_4_monitoring",
                ],
                "slidersArray": Array [
                  "Timing",
                  "Timing/Force",
                  "Spatial",
                  "Temporal",
                  "Control (Postural; FF; FB)",
                  "Monitoring",
                ],
                "title": "Execution",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": "sen_5_oral,sen_5_ocular,sen_5_postural,sen_5_limb,sen_5_gross,sen_5_fine,sen_5_tool,sen_5_projected",
              "nodeType": "function",
              "props": Object {
                "databaseNamesArray": Array [
                  "sen_5_oral",
                  "sen_5_ocular",
                  "sen_5_postural",
                  "sen_5_limb",
                  "sen_5_gross",
                  "sen_5_fine",
                  "sen_5_tool",
                  "sen_5_projected",
                ],
                "slidersArray": Array [
                  "Oral",
                  "Ocular",
                  "Postural",
                  "Limb",
                  "Gross",
                  "Fine",
                  "Tool/Object",
                  "Projected Action Sequences",
                ],
                "title": "Skill Domain",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": "sen_6_somatodyspraxia,sen_6_visual,sen_6_generalized,sen_6_bmc,sen_6_praxis",
              "nodeType": "function",
              "props": Object {
                "databaseNamesArray": Array [
                  "sen_6_somatodyspraxia",
                  "sen_6_visual",
                  "sen_6_generalized",
                  "sen_6_bmc",
                  "sen_6_praxis",
                ],
                "slidersArray": Array [
                  "Somatodyspraxia",
                  "Visual Dyspraxia",
                  "Generalized Dyspraxis",
                  "BMC/Balance/Sequencing/ML",
                  "Praxis on Verbal Command",
                ],
                "title": "Ayres' Factors",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
